feat: add prop to override archive button, export icon (#8)
knocklabs/react-notification-feed: README.md

      userToken={currentUser.knockUserToken}
      // Optionally you can scope the feed in a particular manner
      // tenant={currentWorkspace.id}

>       // Optionally you can make provider to not render any markup and later use KnockFeedContainer to wrap components

Reviewer: ```suggestion
      // Optionally you can stop the provider rendering any markup and use `KnockFeedContainer` to wrap components
```
